metaflow/plugins/datatools/s3/s3.py

```diff
@@ -495,6 +495,10 @@ class S3(object):
 
     @classmethod
     def get_root_from_config(cls, echo, create_on_absent=True):
+        if DATATOOLS_S3ROOT is None:
```

Comment:
@tfurmston any insights into why we need to throw an error here?

Reply:
Yes, I believe it is because this mwthod is called from `IncludeFile` class [here](https://github.com/Netflix/metaflow/blob/master/metaflow/includefile.py#L374). 

@romain-intel mentioned it in the old PR

Reply:
@savingoyal @romain-intel Any thoughts on this one? Happy to remove this line, if you prefer?

Reply:
OK, cool. 

Yeah, it makes sense. I check the `IncludeFile` class only, from which it wasn't obvious to me that any such previous checks would occur. 

I've removed this part from the PR now.